modules/doc/content/newsletter/2023/2023_11.md

```diff
@@ -91,4 +125,36 @@ when transferring between applications.
 
 ## PETSc-level Changes
 
+### PETSc updated to 3.20.1
```

Comment:
Couldn't help it huh? :D

Reply:
Nope 😄 It was either delete the section or add some content....